db/db_impl/db_impl_secondary.cc

```diff
@@ -277,17 +276,13 @@ Status DBImplSecondary::RecoverLogFiles(
       // needed for secondary instances
       if (status.ok()) {
         for (const auto id : column_family_ids) {
-          ColumnFamilyData* cfd =
-              versions_->GetColumnFamilySet()->GetColumnFamily(id);
+          auto cfd = versions_->GetColumnFamilySet()->GetColumnFamily(id);
           if (cfd == nullptr) {
             continue;
           }
-          std::unordered_map<ColumnFamilyData*, uint64_t>::iterator iter =
-              cfd_to_current_log_.find(cfd);
-          if (iter == cfd_to_current_log_.end()) {
-            cfd_to_current_log_.insert({cfd, log_number});
-          } else if (log_number > iter->second) {
-            iter->second = log_number;
+          auto iterbool = cfd_to_current_log_.insert({cfd, log_number});
```

Comment:
This isn't my favorite name, but I can see it's hard to describe "((cfd and current log) and inserted)" in a succinct way.